azure-pipelines.yml

```diff
@@ -26,18 +26,10 @@ resources:
 
 variables:
   - template: "/miscellaneous/build_templates/variables.yml@templates"
```

Comment:
This line and one or two others are the only remaining double quoted path names now that this PR flipped a few others while changing config to appsettings. For styling's sake, do you want to flip this to single quotes even though the path is unchanged?